controllers/module_reconciler.go

```diff
@@ -234,7 +234,7 @@ func (r *ModuleReconciler) handleBuild(ctx context.Context,
 	logger := log.FromContext(ctx).WithValues("kernel version", kernelVersion, "image", km.ContainerImage)
 	buildCtx := log.IntoContext(ctx, logger)
 
-	buildRes, err := r.buildAPI.Sync(buildCtx, *mod, *km, kernelVersion)
+	buildRes, err := r.buildAPI.Sync(buildCtx, *mod, *km, kernelVersion, true)
```

Comment:
If this value is always `true`, what's the point of allowing to push or not the image?
The image will always be pushed, unless I'm missing something else :thinking: